### PR TITLE
Mount Path File Fixes

### PR DESCRIPTION
**Pull request by krishchow**
This PR aims to achieve the following goals:

1. Update the imports for the API, referring to the kube-sigs/cosi-api repository instead of cosi/api repository
2. Update the files created on the pod to include the bucket.json and credentials file
3. Adds the check to validate that the previously mentioned files do not already exist

### CODE REVIEW

**Comment by brahmaroutu** [comment]
@krishchow Please squash the commits.

File: go.mod [hunk 1]
```diff
@@ -3,20 +3,19 @@ module github.com/kubernetes-sigs/container-object-storage-interface-csi-adapter
 go 1.14
```
Comment (brahmaroutu) [inline go.mod]: Can you bump the go version to 1.15

**Comment by krishchow** [comment]
Squashed commits and bumped version @brahmaroutu

**Pull request merged**